### ISSUE TITLE
API: download `.csv` search/traffic analytics

### ISSUE THREAD

**Issue by ericsnekbytes** [readthedocs/readthedocs.org#11362]
Jupyter uses traffic and search metrics CSV's from the admin panel, and they are currently downloaded via a manual process. Allowing automations to authenticate and download this information would reduce the complexity of building automated analytics with ReadTheDocs data.

**Comment by ericsnekbytes** [comment]
@humitos Yes, specifically under Admin > Traffic Analytics / Search Analytics. Thanks!

**Comment by astrojuanlu** [comment]
We would love to have programmatic access to RTD analytics as well 👍🏼
